app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -16,6 +16,8 @@ def index
                             client_id: @authorize_form.client_id,
                             errors: @authorize_form.errors.messages)
 
+      return create if already_allowed?
```

Comment:
Yup! It was a question from one of the CBP developers, and we thought it made sense